ManageWikiNamespaces.php

```diff
@@ -121,6 +121,22 @@
 		'help' => '',
 		'requires' => [],
 	],
+	'wgCosmosRailBlacklistedNamespaces' => [
+		'name' => 'Disable Cosmos side rail in this namespace.',
+		'from' => 'cosmos',
+		'type' => 'check',
+		'main' => true,
+		'talk' => true,
+		'excluded' => [],
+		'overridedefault' => [
+			-1 => true,
```

Comment:
I'm actually not certain how this will work, since NS_SPECIAL isn't available to be modified via MWN. We should consider supporting it actually, while only allowing additional settings to be set for it. We could also consider adding this particular config to ManageWikiSettings, using the `namespaces` type, but I guess this can be tested first.